### PR TITLE
Feature/df 410 form reset

### PR DESCRIPTION
**Pull request by pavax**

### CODE REVIEW

File: alv-portal-ui/src/app/job-ad-search/state-management/actions/job-ad-search.actions.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { JobSearchFilter } from '../state/job-search-filter.types';
 export const INIT_RESULT_LIST = 'JOBS:INIT_RESULT_LIST';
 export const FILTER_APPLIED = 'JOBS:FILTER_APPLIED';
 export const APPLY_FILTER = 'JOBS:APPLY_FILTER';
+export const RESET_FILTER = 'JOBS:RESET_FILTER';
+/*export const RESET_FILTER_APPLIED = 'JOBS:RESET_FILTER_APPLIED';*/
```
Comment (ganqqwerty) [inline alv-portal-ui/src/app/job-ad-search/state-management/actions/job-ad-search.actions.ts]: ;)

File: alv-portal-ui/proxy.conf.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-      "secure": false,
-      "loglevel": "debug"
-    },
-    "/referenceservice/api/*": {
-      "target": "https://staging.job-room.ch",
-      "secure": false,
-      "loglevel": "debug"
-    },
+  "/api/*": {
+    "target": "https://staging.job-room.ch",
```
Comment (99999989) [inline alv-portal-ui/proxy.conf.json]: Is this change on purpose?

Comment (pavax) [reply alv-portal-ui/proxy.conf.json]: reverted it

File: alv-portal-ui/src/app/job-ad-search/state-management/actions/job-ad-search.actions.ts (hunk 1 above)
Comment (pavax) [reply alv-portal-ui/src/app/job-ad-search/state-management/actions/job-ad-search.actions.ts]: removed it
